adapters/messaging-adapter-generator/src/test/java/uk/gov/justice/raml/jms/core/JmsEndpointGeneratorTest.java

```diff
@@ -547,6 +549,51 @@ public void shouldCallJmsProcessorWhenOnMessageIsInvoked() throws Exception {
         verify(dispatcher).dispatch(envelope);
     }
 
+    @Test
+    public void shouldCreateDurableTopicSubscriber() throws Exception {
+        generator.run(raml()
+                        .withBaseUri("message://event/listener/message/people")
+                        .with(resource()
+                                .withRelativeUri("/people.events")
+                                .with(action(POST, "application/vnd.context1.events.abc+json")))
+                        .build(),
+                configurationWithBasePackage(BASE_PACKAGE, outputFolder));
+
+        Class<?> clazz = compiler.compiledClassOf(BASE_PACKAGE);
+        ActivationConfigProperty[] activationConfig = clazz.getAnnotation(MessageDriven.class).activationConfig();
+        assertThat(activationConfig, hasItemInArray(
+                allOf(propertyName(equalTo("destinationType")), propertyValue(equalTo("javax.jms.Topic")))));
```

Comment:
that would make the test coupled to implementation, you normally don't wanna do it